index.js

```diff
@@ -1,7 +1,23 @@
 var NgFaker = require('./lib');
 
+const {
+  en,
+  yo,
+  ig,
+  ha
+} = require('./lib/locales');
+
 // create new instance of faker class
-var ngfaker = new NgFaker({ locales: require('./lib/locales') });
+var ngfaker = new NgFaker({
```

Comment:
Just looked through this @mykeels 

I'm thinking, at some point the data sets is going to get larger as we might be providing more data apart from `title`, `name` and `lorem`...

Won't this be a problem at that point. 

Also, does this mean that when one requires the package all locales will be used at the same time.

Reply:
Hey @mykeels , Looking through this again is there a way we can set the default local to english and only have the user add the locales when they want.

Reply:
Wow, I'd forgotten about this PR 😄... Can we can look through it together tomorrow?

Reply:
Yaaaass!! Legend..